### DISCUSSION TITLE
4주차 스터디 회고

### DISCUSSION THREAD

**Discussion by alanhakhyeonsong** [Next-Level-Study/effective-java-study#14] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by alanhakhyeonsong** [comment]
이번주는 뭔가 스스로 미흡한 것 같네요. 더 잘 준비해서 오겠읍니다...

**Comment by JeongJaew0n** [comment]
지난 주에 코드를 보면서 공부하자고 생각했습니다. 그래서 이번에는 책에도 코드가 많이 나와 있고 실제 코드와 api를 같이 찾아보니 어떻게쓰는지 '방법'은 더 쉽게 이해했습니다. 하지만 그 코드를 '왜' 썼는지에 대한 이해가 완전히 된 건 아니여서 더 생각해보고, 생각한걸 공유할 수 있도록 노력해보겠습니다.

**Comment by sdh46** [comment]
이번 주는 분량이 적고 상대적으로 내용이 쉬운 편이라서 여유가 있었던 것 같습니다. 그리고 궁금한 것에 대한 자료는 찾아 봤지만 찾아본 자료에 대해 한번 더 깊게 보는 연습을 해야겠습니다.

**Comment by narangd0** [comment]
정적 멤버 클래스와 비정적 멤버 클래스의 차이에 대해 논의하는 시간이 되어 유익했다.
4주차쯤 되니 조금 안정 + 적응이 된 것 같다.
다음주 2 ~ 4장 복습에 대해 스터디 어떻게 진행할 지 정해야 할 듯

**Comment by SuJeong68** [comment]
애매한 단어를 많이 이해했다고 생각했는데 아직도 모호하게 받아들이고 있었던 것 같습니다. 더 깊이 공부해야겠습니다.

**Comment by Kurmarkpzgr** [comment]
방법론적인 부분은 이해하였으나, 그 구현은 아직 잘 모르는것 같습니다.
또한 내주에 이전에 한 내용들을 복습, 회고하기로 하였는데, 어떤 식으로 준비 해야하는지 잘 모르겠습니다.

**Comment by opgumo3** [comment]
책 안의 내용을 이해하면서, 저자가 무슨 말을 하고 싶은지  파악을 하는 것이 중요한데 쉽지 않은 것 같습니다.
